packages/docs/content/components/alert.mdx

```diff
@@ -182,23 +182,42 @@ Used to show important but not as urgent messages as standard alerts.
 - Do not visually hide alert messages on the page and then make them visible when they are needed. Users of older assistive technologies may still be able to perceive the alert messages even if they are not currently applicable.
-- Use the <abbr title="Accessible rich internet applications">ARIA</abbr> `role="alert"` to inform assistive technologies of a time-sensitive and important message that is not interactive. If the message is interactive, use the `role="alertdialog"` instead. Both include an implicit use of `aria-live=”assertive”`. Refer to the [Dialog component’s full documentation](/components/modal-dialog/) when using `role=”alertdialog”`.
+- Use the <abbr title="Accessible rich internet applications">ARIA</abbr> `role="alert"` to inform assistive technologies of a time-sensitive and important message that is not interactive. If the message is interactive, use the `role="alertdialog"` instead. Both include an implicit use of `aria-live="assertive"`. Refer to the [Dialog component’s full documentation](/components/modal-dialog/) when using `role="alertdialog"`.
 - If you have multiple alert components on a page, tailor the accessible text to differentiate between each by using an `aria-label` or `aria-labelledby` attribute.
+- The element referenced by the `aria-describedby`
 
 <ThemeContent onlyThemes={['healthcare']}>
 
 #### For dynamic alerts
 
 - If dynamically loaded, be mindful of when the alert displays on the page. For example, an alert could appear on a page due to an error or progressive disclosure. Where and how the content appears visually should align with how and when a screen reader user hears the content. Errors need immediate attention, but content through progressive disclosure may not.
-- For dynamic alert content that is injected into the page when it changes, use the `aria-live` attribute, set on an empty element available on page load. The value of the `aria-live` attribute is used to update assistive technologies on the degree of importance.
-  - `aria-live=”polite”` will notify users of updates but generally do not interrupt the current task, with the updates having a low priority.
-  - `aria-live=”assertive”` will immediately notify the user, potentially clearing the speech queue of previous updates. Don't use the assertive value unless the interruption is imperative
+- When a user's actions require dynamically displaying an alert, such as with success or error states, inject the alert inside an already existing `<div>` with `aria-live="polite"` and `aria-relevant="additions removals"` or `role="alert"`. These ARIA attributes ensure screen readers will announce alerts that have been added or updated dynamically.
+
+  - For example, your empty `<div>` on page load would appear as:
+
+  ```html
+  <div aria-live="polite" aria-relevant="additions removals"></div>
+  ```
+
+  And whn the user performs an action that displays an alert:
```

Comment:
"whn" - typo

I also think this sentence could end with a clause about the action instead of just relying on the code below, like "And when the user performs an action that displays an alert, add the alert to the existing div:" I know it's redundant with the explanation above, but I think that parallelism would reinforce the concept.